source/tutorials/packaging-existing-software.md

```diff
@@ -5,7 +5,7 @@ myst:
     "keywords": "Nix, packaging"
 ---
 
-(packaging-existing-software)=
+(packaging-tutorial)=
```

Comment:
This is somewhat unrelated, because preserving the URL means we have to add a redirect from summer.nixos.org. @delroth (as our infra liaison) where would that be defined?

Reply:
I am not talking about preserving the URL of the [original post written during SoN '22](https://summer.nixos.org/blog/callpackage-a-tool-for-the-lazy/), but about the URL of [packaging existing software tutorial on nix.dev](https://nix.dev/tutorials/packaging-existing-software) whichs URL you change here in this hunk.

Reply:
Oh no, that's just an internal identifier for cross-references. Changing an URL would mean moving the file, and we'd always put in a redirect in that case.

Reply:
I misunderstood the purpose of that alias.

Thanks for clarifying!